from Core.ExceptionsRegistry import RegisterExc, NetException


class TestError(NetException):
    pass